symengine/series_flint.cpp

```diff
@@ -138,8 +138,8 @@ unsigned URatPSeriesFlint::ldegree(const fqp_t &s)
 {
     long i = 0;
     while (i <= s.degree())
-        if (not s.get_coeff(i++).is_zero())
-            return i - 1;
+        if (not((s.get_coeff(i++)).is_zero()))
```

Comment:
@isuruf This line is giving away this error:
```
error: conversion to ‘unsigned int’ from ‘long int’ may alter its value [-Werror=conversion]
```
even though `is_zero()` would return a `bool`. Any idea why is this cropping up?